conf/zeppelin-env.cmd.template

```diff
@@ -17,7 +17,7 @@ REM limitations under the License.
 REM
 
 REM set JAVA_HOME=
-REM set MASTER=                 		REM Spark master url. eg. spark://master_addr:7077. Leave empty if you want to use local mode.
+REM set SPARK_MASTER=                 		REM Spark master url. eg. spark://master_addr:7077. Leave empty if you want to use local mode.
```

Comment:
Not aligned with other rows

Reply:
yeah, the both files contained a mix of tabs & spaces, why it wasn't catch up...
Fixed in both